Sources/FunctionalCss/GridHelpersUpgrade.swift

```diff
@@ -3,33 +3,22 @@ import HtmlUpgrade
 
 // TODO: extract to grid helpers in design systems?
 extension Node {
-  public static func _gridRow(_ attribs: [Attribute<Tag.Div>], _ content: [Node]) -> Node {
-    return .div(attributes: _addClasses([Class.grid.row], to: attribs), .fragment(content))
+  public static func gridRow(attributes: [Attribute<Tag.Div>] = [], _ content: Node...) -> Node {
+    return .div(attributes: _addClasses([Class.grid.row], to: attributes), .fragment(content))
   }
 
-  // TODO: extract to grid helpers in design systems?
-  public static func _gridRow(_ content: [Node]) -> Node {
-    return ._gridRow([], content)
-  }
-
-  // TODO: extract to grid helpers in design systems?
-  public static func _gridColumn(sizes: [Breakpoint: Int]) -> ([Node]) -> Node {
```

Comment:
Removed this curried overload for now...assuming we don't want it.

Reply:
yeahhhh there are maybe a few uses of it but we can just remove as we come across them

Reply:
ha let's revisit it when we need it.